### PR TITLE
Warn not to do global operations before upgrading all CRDBs

### PR DESCRIPTION
**Pull request by Paz-Yanover**
valentino@redislabs.com, @bmansheim - please review and approve

### CODE REVIEW

**Comment by bmansheim** [comment]
@Paz-Yanover Please review my commit

**Comment by bmansheim** [comment]
@Paz-Yanover Can you make an alternative suggestion?

**Comment by Paz-Yanover** [comment]
@bmansheim - see https://github.com/RedisLabs/redislabs-docs/pull/559/commits/3c006a4b6e7c4fff29b359c676e8b7305736f72d

**Comment by bmansheim** [comment]
@Paz-Yanover We want to have the note higher up on the page to make sure customers see it, and the two ideas are similar. How does it look now?

**Comment by Paz-Yanover** [comment]
@bmansheim - I don't think the sentence is correct, as the second point is not "to avoid temporary inconsistencies between the instances".
@valentino@redislabs.com - what do you think?

**Comment by Paz-Yanover** [comment]
@bmansheim - Valentino agrees with my comment above

**Comment by Paz-Yanover** [comment]
@bmansheim -
Let's add the following:
{{% note %}}
Until upgrading all instances of a specific CRDB, avoid performing global operations on this CRDB, like removing instance, adding new instance etc.
{{% /note %}}

File: content/rs/installing-upgrading/upgrading.md [hunk 1]
```diff
@@ -130,8 +130,8 @@ instances that were not upgraded yet cannot receive write updates from the upgra
 The upgraded instance receives updates from upgraded and non-upgraded instances.
 
 {{% note %}}
-- Upgrade all instances of a specific CRDB within a reasonable time frame to avoid temporary inconsistencies between the instances.
-- Make sure that you upgrade all instances of a specific CRDB before you do global operations on the CRDB such as removing instances and adding new instances.
+- Upgrade all instances of a specific CRDB within a reasonable time frame.
```
Comment (Paz-Yanover) [inline content/rs/installing-upgrading/upgrading.md]: @bmansheim - why did you remove the "reason" here?
